src/psyclone/domain/lfric/lfric_invoke.py

```diff
@@ -80,7 +80,6 @@ def __init__(self, alg_invocation, idx, invokes):
         reserved_names_list = []
         const = LFRicConstants()
         reserved_names_list.extend(const.STENCIL_MAPPING.values())
-        reserved_names_list.extend(const.VALID_STENCIL_DIRECTIONS)
```

Comment:
Now these are handled by the PSyIR (specifically in the kernel call lowering), and to always refer to the same symbol it uses tags.

I believe the reserved_name_list is because some these are/were handled as strings produced by gen_code and we don't want to have symbols and gen_code strings with the same name, but we can get rid of all this when everything is in PSyIR.